mockResponse.json

```diff
@@ -0,0 +1,38 @@
+{
+    "name": "Rohini",
+    "email": "roh3290a@gmail.com",
+    "password": "HalogenIsTheBest2021",
+    "new-user": "true",
```

Comment:
We can actually use something called local storage in the browser to really easily tell if a user has visited the site before or not, so we can remove the `new-user` property here